pyfolio/perf_attrib.py

```diff
@@ -35,51 +39,11 @@ def perf_attrib(returns,
                 transactions=None,
                 pos_in_dollars=True):
     """
-    Does performance attribution given risk info.
+    Preprocesses inputs and then calls ep.perf_attrib.
 
     Parameters
     ----------
-    returns : pd.Series
```

Comment:
I would probably still leave these docstrings here.  There's a little bit of a risk that these might get out of date if we make changes in empyrical, but I think it's valuable for a pyfolio user to be able to read these without having to dig through a separate library.  IMO, a user that's using basic pyfolio functionality shouldn't have to know or care about the fact that empyrical exists; it should be mostly an implementation detail of the fact that we need to share code between projects.

Reply:
At some point we might also want to consider something like [`zipline.utils.sharedoc.templated_docstring`](https://github.com/quantopian/zipline/blob/master/zipline/utils/sharedoc.py#L85-L101) for doc chunks that we want to share across libraries.